packages/kbn-monaco/src/console/lexer_rules/shared.ts

```diff
@@ -100,17 +107,22 @@ xjsonRules.json_root = [
   matchToken('variable.template', /("\${\w+}")/),
   // @ts-expect-error include a rule to start sql highlighting
   buildSqlStartRule(),
+  // @ts-expect-error include a rule to start sql highlighting
```

Comment:
nit: shouldn't this comment be to start painless highlighting?

Reply:
Nice catch! Fixed with [18a2f1f](https://github.com/elastic/kibana/pull/202695/commits/18a2f1fbc54d01faa2538ae8c27fe5b8d4e518ed). Thanks for the review!